### ISSUE TITLE
New resource group for urbanroute

### ISSUE THREAD

**Issue by PatrickOHara** [alan-turing-institute/urbanroute#12]
**Is your feature request related to a problem? Please describe.**

We will need a small cluster of VMs and a mini API to interact with the frontend.

**Describe the solution you'd like**

Create a new Azure resource group for routing.

Function:
- Run cron jobs for creating materialized view as in #11 
- Run routing algorithms when a user requests a route. May require multiple low cost VMs to handle concurrent algorithms.
- Power a mini API for routing. This would hide behind our main API for security and modularity purposes.

It would need read access to use certain schemas within the cleanair database
- [ ] static data schema for the OSHighways tables
- [ ] air quality modelling schema to access model results.
- [ ] interest points schema for the meta points table.

Why create a new resource group?
- Routing is not really part of the cleanair infrastructure, it makes sense to separete it
- Security: no need for routing to have access to e.g. scoot data
- If the routing component becomes bigger, it will be easier to separete it in future.

Disadvantages
- Potentially more maintenence

**Describe alternatives you've considered**

The alternative is to keep everything within the same resource group. This seems messy and would need close intergration with the cleanair repo.

**Additional context**

A diagram might help:

![image](https://user-images.githubusercontent.com/25480498/85203108-37c2ab80-b303-11ea-9ff4-5d6b6c739134.png)

@OscartGiles @JamesCraster thoughts?

**Comment by OscartGiles** [comment]
Could all run on the existing cluster. Would you like a separate API? Or are you happy to add to the existing one?

**Comment by OscartGiles** [comment]
I would say we should use the same cluster to keep maintenance low. But we can use a different namespace on the cluster to separate it from the other stuff.

**Comment by PatrickOHara** [comment]
Yes happy to run on existing cluster with a different namespace. Thats makes sense

It makes sense to add to the existing API. I was wondering if the `urbanair` API could do the security and handle the basic requests then "point" to the `urbanroute` module?
